backend/app/Contracts/Visitors/VisitorsBounceRateByParameter.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Contracts;
+
+interface VisitorsBounceRateByParameter
```

Comment:
Not quite sure if you need interface, it seems you have only one implementation, and always will have only one. (ignore this comment if it's common practice in project))

Reply:
Yes, it is not required to create interface for one implementaion. We have some general interfaces for visitors api, because there should be some contract for responses, as they all similar. It is not required to get rid of that, but pay attention to this comment in future